test/suites/auth.sh

```diff
@@ -49,7 +49,11 @@ test_authorization() {
   lxc auth group permission remove test-group instance c1 can_exec project=default # Valid
   ! lxc auth group permission remove test-group instance c1 can_exec project=default || false # Already removed
   ! lxc auth group permission add test-group instance c1 not_an_instance_entitlement project=default || false # Invalid entitlement
+
+  # Test permission is removed automatically when instance is removed.
+  lxc auth group permission add test-group instance c1 can_exec project=default # Valid
   lxc rm c1
```

Comment:
Please always add the `--force` flag to the test suite to ensure if it is running it wont hang the suite.